### DISCUSSION TITLE
Feature Request: Support Cockroachdb (written in Go) global serverless db as a secondary option

### DISCUSSION THREAD

**Discussion by algoflows** [pocketbase/pocketbase#868] [Q&A]
CockroachDB written in Go as a secondary option for PocketBase, bringing scalability and performance benefits of a true  serverless globally distributed database with acid transactions and global horizontal scaling. 

https://www.cockroachlabs.com/

**Comment by ganigeorgiev** [comment]
Support for other SQL database may be added in the future but for now it's has the lowest priority. You could check this discussion - https://github.com/pocketbase/pocketbase/discussions/14#discussioncomment-3109447

**Comment by algoflows** [comment]
This feature request specifically pertains to Cockroachdb and shouldn't be closed.

**Comment by algoflows** [comment]
Surely it would be great to see what kind of response the community has for this request??

**Comment by algoflows** [comment]
Not super happy about how quickly you've closed the feature request as its not been requested anywhere else.

**Comment by ganigeorgiev** [comment]
I don't plan working on this for now and there is no point with bloating the issues tracker. We have a roadmap that you could check at https://github.com/orgs/pocketbase/projects/2.

Support for other databases may be added not because of the eventual scalability benefits (the sqlite db is not the only reason why PocketBase cannot scale horizontally), but mainly because of servers with ephemeral storage.

_I'll move the issue to Q&A discussion._
